jobs/src/main/scala/dcos/metronome/repository/LoadContentOnStartup.scala

```diff
@@ -45,6 +48,24 @@ trait LoadContentOnStartup[Id, Model] extends Actor with Stash with ActorLogging
         throw ex
     }
   }
+
+  private def getModel(id: Id): Future[Option[Model]] = {
+    repo.get(id).recoverWith {
+      case ex: StoreCommandFailedException =>
+        ex.getCause match {
+          case cause: InvalidProtocolBufferException =>
+            log.error(s"ID $id has corrupt data and will be ignored.  Zk will need to be manually repaired.  Exception message: ${cause.getMessage}")
+            Future.successful(None)
```

Comment:
I think that we should throw an exception in this case, don't you?

Reply:
@timcharper  No!   We can't!   This is in the PreStart of the Actor... the exception is what is taking out the Actor and causing the 500 errors.      this approach shouldn't be an issue... there will be no data loss and they can't write over the existing node.

Reply:
I'm really confused. Why wouldn't we crash in this case, if zookeeper data is detected to be corrupt?

Reply:
> Seems appropriate for `log.error(ex, "Can not load initial data. Give up.")` to end up with a process abortion.   The invalid state of zk should have completely stopped the process.

Reply:
that is an older message.. and may apply sometimes... you and I discussed this and I thought we landed here...

Reply:
my bad...   we do want to die for corrupt data...   we want to prune for no node expection... I'll have an update shortly